src/database/seeders/20220323100028-demo-accomodation.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable strict */
+/* eslint-disable import/no-import-module-exports */
+
+'use strict';
+
+module.exports = {
```

Comment:
```suggestion
async up (queryInterface, Sequelize) {
      await queryInterface.bulkInsert(
        'accomodations',
        [
          {
            id: 3,
            name: 'Serena Hotel',
            location: 'Rubavu',
            description: 'This is a very prestigious hotel',
            latitude:'234234',
            longitude: '234324',
            userId: 'b66cfc7c-be2c-41f5-b459-e888bfe881a6',
            managerId:'d53e3469-661e-40c0-a2bb-d66e0e5fa19d',
            createdAt: new Date(),
            updatedAt: new Date()
          },
  
          {
            id: 2,
            name: 'Silent Hill',
            description: 'This is a very silent hotel',
            latitude:'234234',
            longitude: '234324',
            location: 'Kayonza',
            userId: 'b66cfc7c-be2c-41f5-b459-e888bfe881a6',
            managerId: '861d7c9d-a8a8-4308-bce8-2c6c6a66c841',
            createdAt: new Date(),
            updatedAt: new Date()
          },
        ],
        {}
      );
  },

  async down(queryInterface, Sequelize) {
    await queryInterface.bulkDelete('accomodations', null, {});
  }
```
@rukundo-kevin use ES6 in your seeders. change it to something similar to the proposed suggestion or refer to the `20220331192923-random-user.js` seeder file for more clarification.